### ISSUE TITLE
Name Property missing for Progress text box present under header 'A determinate Progress bar'.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#325]
**Describe the bug**

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery.
2. Activate ProgressBar from home page.
3. Navigate to text box under header A determinate progress bar

**Expected behavior**

Name Property should be there for text box,

**Actual behavior**

Text box name property missing.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/72557918-8781e000-3856-11ea-96dc-edde20bcbf91.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.12.0

**Additional context**

Copied from [internal bug](https://dev.azure.com/microsoft/OS/_workitems/edit/24649907)

**Comment by chingucoding** [comment]
I think this is an issue with the NumberBox control, since setting AutomationPropterties.Name or AutomationProperties.LabeledBy both do not affect the TextBox but rather the "spinner" that is listed in Accessibility Insights.
